content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -94,23 +96,79 @@ Requires `dd-trace-php` version 0.71+.
 
-The values on the left side is the time spent in that method call during the selected span. Depending on the runtime and language, this list of types varies:
-
-- **Method durations** shows the overall time taken by each method from your code.
+The values on the left side is the time spent in that method call during the selected span. Depending on the runtime and language, the categories vary:
+{{< programming-lang-wrapper langs="java,python,go,ruby,dotnet,php" >}}
+{{< programming-lang lang="java" >}}
 - **CPU** shows the time taken executing CPU tasks.
 - **Synchronization** shows the time spent waiting on monitors, the time a thread is sleeping and the time it is parked.
-- **VM operations** (Java only) shows the time taken waiting for VM operations that are not related to garbage collection (for example, heap dumps).
+- **VM operations** shows the time taken waiting for VM operations (garbage collections, compilation, safepoints, heap dumps, ...).
 - **File I/O** shows the time taken waiting for a disk read/write operation to execute.
 - **Socket I/O** shows the time taken waiting for a network read/write operation to execute.
 - **Monitor enter** shows the time a thread is blocked on a lock.
 - **Uncategorized** shows the time taken to execute the span that cannot be placed into one of the above categories.
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+- **CPU** shows the time taken executing CPU tasks.
+- **Lock Wait** shows the time a thread is blocked on a lock.
+- **Uncategorized** shows the time taken to execute the span that cannot be placed into one of the above categories.
+{{< /programming-lang >}}
+{{< programming-lang lang="ruby" >}}
+- **CPU** shows the time taken executing CPU tasks.
+- **Uncategorized** shows the time taken to execute the span that is not CPU execution.
+{{< /programming-lang >}}
+{{< programming-lang lang="go" >}}
+- **CPU** shows the time taken executing CPU tasks.
+- **Off-CPU** shows the time taken to execute the span that is not CPU execution.
+{{< /programming-lang >}}
+{{< programming-lang lang="dotnet" >}}
+- **CPU** shows the time taken executing CPU tasks.
+- **Lock Wait** shows the time a thread is blocked on a lock.
+- **Uncategorized** shows the time taken to execute the span that cannot be placed into one of the above categories.
+{{< /programming-lang >}}
+{{< programming-lang lang="php" >}}
+- **CPU** shows the time taken executing CPU tasks.
+- **Uncategorized** shows the time taken to execute the span that is not CPU execution.
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
 
 Click the plus icon `+` to expand the stack trace to that method **in reverse order**. Hover over the value to see the percentage of time explained by category.
 
+### Span execution timeline view
+
+{{< img src="profiler/code_hotspots_tab-timeline.mp4" alt="Code Hotspots tab has a timeline view that breakdown execution over time and threads" video=true >}}
```

Comment:
🐞  This link doesn't seem to work.

![CleanShot 2023-07-05 at 13 23 56@2x](https://github.com/DataDog/documentation/assets/15000/538d6bee-6acb-473b-9df4-7cefb7375546)

Reply:
I believe this is an issue with the CI build. In the meantime, you can download it in the PR 😅

Reply:
I saw this problem in my local build. I think the filename is wrong. There is no file called `code_hotspots_tab-timeline.mp4`:

<img width="651" alt="CleanShot 2023-07-06 at 08 53 36@2x" src="https://github.com/DataDog/documentation/assets/15000/0d93113b-540b-4d5c-aae3-ae786cd44803">

Reply:
Fixed in https://github.com/DataDog/documentation/pull/18788/commits/fcac5ba7582ffd42b471e9dad7ce00f427a18657, thanks!